tests/integration/test_migrate.sh

```diff
@@ -72,7 +72,7 @@ trap cleanup EXIT
 # at /var/lib/docker
 atomic_migrate () {
 	setup
-	${ATOMIC} migrate export --dir "$(pwd)/migrate-dir"
+	echo 'y'|${ATOMIC} migrate export --dir "$(pwd)/migrate-dir"
```

Comment:
We are adding a --quiet in a different pull request.  I am not sure I have ever seen a --yes option on a command.